load_latin_gloss.py

```diff
@@ -15,13 +15,20 @@
     batch = []
     all_lemmas_qs = Lemma.objects.all()
     all_gloss_qs = Gloss.objects.all()
+    lemmas_data = {q.lemma: q for q in all_lemmas_qs}
+    all_gloss_data = {str(g.lemma_id) + g.gloss: g for g in all_gloss_qs}
```

Comment:
@arthurian not sure what effect it would have on speed, but you could offload some memory requirements onto the ORM using:

```py
    all_lemmas_qs = Lemma.objects.all().values()  # need all fields on line 31
    all_gloss_qs = Gloss.objects.all().values('lemma_id', 'gloss')  # maybe only need these two fields?
```

[Using `values()` would return `dict`s instead of entire ORM objects](https://docs.djangoproject.com/en/dev/ref/models/querysets/#django.db.models.query.QuerySet.values), which might give you a speed and memory boost as well.